write-fonts/src/tables/glyf.rs

```diff
@@ -430,6 +460,35 @@ mod tests {
         path.0
     }
 
+    // For `indexToLocFormat == 0` (short version), offset divided by 2 is stored, so add a padding
+    // byte if the length is not even to ensure our computed bytes match those of our test glyphs.
+    fn pad_for_loca_format(loca: &read::tables::loca::Loca, mut bytes: Vec<u8>) -> Vec<u8> {
+        if matches!(loca, read::tables::loca::Loca::Short(_)) && bytes.len() & 1 != 0 {
+            bytes.push(0);
+        }
+        bytes
+    }
```

Comment:
@dfrg this shouldn't be necessary? The `FontWrite` impl for `SimpleGlyph` should handle it already.

Reply:
I may have missed that but when I ran tests they were failing with a missing padding byte when the length of the glyph data was odd.

Reply:
interesting, that would indicate a bug on my part I think!

Reply:
I assumed the adjustment wasn’t made because it ultimately depends on whether short or long offsets are required and we need the full set of compiled glyphs to make that determination